src/sentry/mediators/external_requests/alert_rule_action_requester.py

```diff
@@ -46,7 +47,7 @@ def _make_request(self) -> Mapping[str, Union[bool, str]]:
                 method=self.http_method,
                 data=self.body,
             )
-        except Exception as e:
+        except RequestException as e:
```

Comment:
Is it a problem that this will now raise an exception if we get something other than `RequestException`?

Reply:
It would already raise an `AttributeError` when it gets to the `e.response` so I think it's fine to let exceptions propagate up.